src/components/search/SearchItems.js

```diff
@@ -0,0 +1,25 @@
+import DisplayImage from '../display/DisplayImage'
+import "./Search.css";
+
+const SearchItems = ({ filteredImage }) => {
+
+const filtered = filteredImage.map(image => <DisplayImage key={image.title} image={image}/>)
```

Comment:
Why bring this out of line 10?

Reply:
removed the file and implemented the search differently.